src/libopensc/cardctl.h

```diff
@@ -350,6 +351,14 @@ typedef struct sc_cardctl_pkcs11_init_pin {
 	size_t			pin_len;
 } sc_cardctl_pkcs11_init_pin_t;
 
+/*
+ * Generic cardctl - card driver can examine token info
+ */
+struct sc_cardctl_parsed_token_info {
+	unsigned int flags;
+	struct sc_pkcs15_tokeninfo * tokeninfo;
```

Comment:
clang-format suggested the following change:


```
-	struct sc_pkcs15_tokeninfo * tokeninfo;
+	struct sc_pkcs15_tokeninfo *tokeninfo;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/cardctl.h -->